src/Blockcore.Indexer.Cirrus/appsettings.json

```diff
@@ -42,6 +42,7 @@
       "AverageInterval": 10,
 
       // Store the trx hex in mongo storage or read it from RPC
-      "StoreRawTransactions": true
+      "StoreRawTransactions": true,
+      "NumberOfPullerTasksForIBD" : 2
```

Comment:
notice there are also this params above
We should check which are not being used and reuse them or delete them
```
    // Syncing parameters
      "SyncBlockchain": true,
      "SyncMemoryPool": true,
      "MongoBatchSize": 10000,
      "ParallelRequestsToTransactionRpc": 50,
      "DetailedTrace": 0,
      "MaxItemsInQueue": 10,
      "SyncInterval": 5,
      "AverageInterval": 10,
```

Reply:
I added new ones but didn't go through all of them to check what is obsolete